code/modules/research/techweb/all_nodes.dm

```diff
@@ -597,11 +597,11 @@
 
 /datum/techweb_node/adv_rcd_upgrade
 	id = "adv_rcd_upgrade"
-	display_name = "Advanced RCD designs upgrade"
+	display_name = "Advanced Bluespace RCD designs."
 	description = "Unlocks new RCD designs."
-	design_ids = list("rcd_upgrade_silo_link")
+	design_ids = list("RCD_bluespace", "rcd_upgrade_silo_link")
```

Comment:
This might be a silly question, but is there a point in having the upgrade design present at this point? The cost of the upgraded RCD is pretty significantly different from the base cost you'd get from having a normal RCD with the upgrade, for a slight time delay benefit.

Reply:
The upgraded tool fits belts as well, but you might rather upgrade your RCD if low on exotic materials